doc/rtd/conf.py

```diff
@@ -18,7 +19,8 @@
 
 # General information about the project.
 project = "cloud-init"
-copyright = "Canonical Ltd."
+author = "Canonical Group Ltd."
+copyright = "%s, %s" % (datetime.date.today().year, author)
```

Comment:
How do we feel about f-strings?
```python
copyright = f"Canonical Group Ltd., {datetime.date.today().year}"
```

Reply:
I'm FOR 'em! (My Python is just a bit rusty)

Reply:
Thanks for the review, I'll wait til Monday before applying it though, just in case anyone hates f-strings with a burning passion.

Reply:
> Thanks for the review, I'll wait til Monday before applying it though, just in case anyone hates f-strings with a burning passion.

It's up to you, but in cloud-init the current devs prefer f-strings in most places (except in [logging messages as an optimization](https://docs.python.org/3/howto/logging.html#optimization)), so I don't think you'll get any push back on that part.

Reply:
Good to know! :D Thanks, I'll get those changes pushed over